src/main/java/com/textmessenger/service/LoginServiceImpl.java

```diff
@@ -0,0 +1,38 @@
+package com.textmessenger.service;
+
+import com.textmessenger.model.entity.dto.LoginRq;
+import com.textmessenger.model.entity.dto.Token;
+import com.textmessenger.security.JwtTokenProvider;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Service;
+
+@Service
+
+public class LoginServiceImpl implements LoginService {
+  private final AuthenticationManager authenticationManager;
+  private final JwtTokenProvider jwtTokenProvider;
+
+  public LoginServiceImpl(AuthenticationManager authenticationManager,
+                          JwtTokenProvider jwtTokenProvider) {
+    this.authenticationManager = authenticationManager;
+    this.jwtTokenProvider = jwtTokenProvider;
+  }
+
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "passwordEncoder" private field. ([squid:S1068](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS1068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=142&issues=AWWFmk07XKENQn1S911i&open=AWWFmk07XKENQn1S911i)

[ISSUE_KEY]: AWWFmk07XKENQn1S911i